pygmt/src/grdfill.py

```diff
@@ -57,6 +63,8 @@ def grdfill(grid, **kwargs):
         - None if ``outgrid`` is set (grid output will be stored in file set by
           ``outgrid``)
     """
+    if "A" not in kwargs.keys() and "L" not in kwargs.keys():
+        raise GMTInvalidInput("At least parameter 'mode' or 'L' must be specified.")
```

Comment:
Can we add a long alias for parameter `-L`? GMT.jl uses `list` (see https://www.generic-mapping-tools.org/GMT.jl/dev/#GMT.grdfill), but probably not a good idea to use it for Python since it's a reserved name. Any good suggestions?

Reply:
Per @seisman's comment above, I thought the plan was to leave `-L` for a separate PR?

Reply:
Oh wow, how did I miss that comment! Ok to just keep it as 'L' here for now then.